src/ray/protobuf/monitor.proto

```diff
@@ -32,6 +32,70 @@ message DrainAndKillNodeReply {
   repeated bytes drained_nodes = 2;
 }
 
+message ResourceBundle {
```

Comment:
Q: Is this the protocol we decide to go? How this is going to work with the design @scv119 is working on?

Reply:
This is just the existing design with a stabilized protobuf. I've removed the unused fields in the process of copying things over. This is just the `Bundle` message from `common.proto` with the uncessary fields thrown out. 

Unless the final new design proposal looks extremely similar to the old one, we'll likely introduce a new endpoint to implement the new design, depending on how similar the design ends up being we may reuse some fields if it makes sense, but it's fine if we don't.